Kata RPG/Kata RPG/Test.swift

```diff
@@ -0,0 +1,90 @@
+//
+//  Test.swift
+//  Kata RPG
+//
+//  Created by norm on 05/11/2020.
+//
+
+import Foundation
+
+let c1 = Character(name: "Link")
+let w1 = Warrior(name: "Ganon")
+let p1 = Priest(name: "Zelda")
+
+func firstIteration(){
```

Comment:
Name the function with what it does. If you can't, it certainly means you should refacto

Reply:
Ok took time but I think I got it